### PR TITLE
[ADD] adding selttment lines tree view with pivot table for analysis …

### PR DESCRIPTION
**Pull request by jackjack82**
…and related "state" field taken from settlement

### CODE REVIEW

**Review by giusygiallara93 — APPROVED** [review]

**Comment by jackjack82** [comment]
could please someone review, approve and merge? thx

File: sale_commission/models/settlement.py [hunk 1]
```diff
@@ -159,4 +159,10 @@ class SettlementLine(models.Model):
     settled_amount = fields.Float(
         related="agent_line.amount", readonly=True, store=True)
     commission = fields.Many2one(
-        comodel_name="sale.commission", related="agent_line.commission")
+        comodel_name="sale.commission", related="agent_line.commission",
+        store=True, string="Commission")
+    state = fields.Selection(
+        related="settlement.state", string="State", store=True)
```
Comment (pedrobaeza) [inline sale_commission/models/settlement.py:165]: `readonly=True` please

File: sale_commission/models/settlement.py [hunk 2]
```diff
@@ -159,4 +159,10 @@ class SettlementLine(models.Model):
...
+    partner_id = fields.Many2one(
```
Comment (pedrobaeza) [inline sale_commission/models/settlement.py:166]: `readonly=True` please

File: sale_commission/i18n/pt_BR.po [hunk 3]
```diff
@@ -621,8 +621,8 @@ msgstr ""
 #. module: sale_commission
 #: model:ir.actions.act_window,name:sale_commission.action_settle_commission
 #: model:ir.ui.menu,name:sale_commission.menu_sattle_commissions
-msgid "Settlments"
-msgstr ""
+msgid "Settlements"
```
Comment (pedrobaeza) [inline sale_commission/i18n/pt_BR.po:624]: Please modify msgid of the rest of the .po files

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
Is this going to be continued?

**Comment by pedrobaeza** [comment]
Closing as no answer.

**Pull request closed without merging**